### PR TITLE
Add Russian translation

### PR DESCRIPTION
**Pull request by lenemter**
And update other extra .po files

### CODE REVIEW

**Review by dar5hak — APPROVED** [review]

File: po/extra/es.po [hunk 1]
```diff
@@ -99,44 +95,54 @@ msgstr ""
...
-#: data/com.github.elfenware.obliviate.appdata.xml.in:33
+#: data/com.github.elfenware.obliviate.appdata.xml.in:27
+#, fuzzy
+msgid "Update: Dutch translations (@Vistaus)"
+msgstr "Adición: traducción en neerlandés (@Vistaus)"
+
+#: data/com.github.elfenware.obliviate.appdata.xml.in:28
+#, fuzzy
+msgid "Update: Italian translations (@albanobattistella)"
+msgstr "Adición: traducción en neerlandés (@Vistaus)"
```
Comment (dar5hak) [inline po/extra/es.po:106]:
```suggestion
msgstr "Adición: traducción en italiano (@ albanobattistella)"
```

File: po/extra/es.po [hunk 2]
```diff
@@ -99,44 +95,54 @@ msgstr ""
...
+
+#: data/com.github.elfenware.obliviate.appdata.xml.in:35
 msgid "Update: Use elementary Platform 7"
 msgstr ""
 
-#: data/com.github.elfenware.obliviate.appdata.xml.in:40
+#: data/com.github.elfenware.obliviate.appdata.xml.in:42
 #, fuzzy
 msgid "Add: Italian translations (@albanobattistella)"
 msgstr "Adición: traducción en neerlandés (@Vistaus)"
```
Comment (dar5hak) [inline po/extra/es.po:115]:
```suggestion
msgstr "Adición: traducción en italiano (@ albanobattistella)"
```

File: po/extra/fr.po [hunk 3]
```diff
@@ -100,43 +96,56 @@ msgstr ""
...
-#: data/com.github.elfenware.obliviate.appdata.xml.in:33
+#: data/com.github.elfenware.obliviate.appdata.xml.in:27
+#, fuzzy
+msgid "Update: Dutch translations (@Vistaus)"
+msgstr "Ajout : Traductions en néerlandais (@Vistaus)"
+
+#: data/com.github.elfenware.obliviate.appdata.xml.in:28
+#, fuzzy
+msgid "Update: Italian translations (@albanobattistella)"
+msgstr "Ajout : Traductions en néerlandais (@Vistaus)"
```
Comment (dar5hak) [inline po/extra/fr.po:107]:
```suggestion
msgstr "Ajout : Traductions en italien (@albanobattistella)"
```

File: po/extra/fr.po [hunk 4]
```diff
@@ -100,43 +96,56 @@ msgstr ""
...
+
+#: data/com.github.elfenware.obliviate.appdata.xml.in:35
 msgid "Update: Use elementary Platform 7"
 msgstr ""
 
-#: data/com.github.elfenware.obliviate.appdata.xml.in:40
+#: data/com.github.elfenware.obliviate.appdata.xml.in:42
 #, fuzzy
 msgid "Add: Italian translations (@albanobattistella)"
 msgstr "Ajout : Traductions en néerlandais (@Vistaus)"
```
Comment (dar5hak) [inline po/extra/fr.po:116]:
```suggestion
msgstr "Ajout : Traductions en italien (@albanobattistella)"
```

**Review by dar5hak — CHANGES_REQUESTED** [review]

**Comment by dar5hak** [comment]
@lenemter Would you also be wiling to add Russian translations for future releases? If you would, I can add you to the translators group, where I can notify you in case of an upcoming update and you can get the translations in before I publish the update. Let me know if you are interested.

(No hard feelings if you don't want to.)

**Comment by lenemter** [comment]
@dar5hak Yes I would. Send me instructions if I need to to do anything

**Comment by dar5hak** [comment]
@lenemter Thank you! I sent you an invite to the team.

**Pull request merged**